acceptance/config/beaker/jenkins/redhat7-64m-64a.cfg

```diff
@@ -0,0 +1,30 @@
+---
+HOSTS:
+  redhat7-64-1:
+    roles:
+    - agent
+    - master
+    hypervisor: vcloud
```

Comment:
This value came from a recent run of `genconfig2` in version 0.12.1:

I think we're on the right path to get rid of these static config files, but perhaps `sqa-utils-gem` needs to be updated as well?

```
$ bundle exec genconfig2 redhat7-64m-64a
---
HOSTS:
  redhat7-64-1:
    roles:
    - agent
    - master
    hypervisor: vcloud
    pe_dir:
    pe_ver:
    pe_upgrade_dir:
    pe_upgrade_ver:
    platform: el-7-x86_64
    template: Delivery/Quality Assurance/Templates/vCloud/redhat-7-x86_64
  redhat7-64-2:
    roles:
    - agent
    hypervisor: vcloud
    pe_dir:
    pe_ver:
    pe_upgrade_dir:
    pe_upgrade_ver:
    platform: el-7-x86_64
    template: Delivery/Quality Assurance/Templates/vCloud/redhat-7-x86_64
CONFIG:
  nfs_server: none
  consoleport: 443
  datastore: instance0
  folder: Delivery/Quality Assurance/Enterprise/Dynamic
  resourcepool: delivery/Quality Assurance/Enterprise/Dynamic
  pooling_api: http://vcloud.delivery.puppetlabs.net/
```